<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>851</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>851</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>851</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pass Paulito Manager v1.0</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/IconImage/Icon.png</normaloff>:/IconImage/Icon.png</iconset>
  </property>
  <property name="toolTip">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="whatsThis">
   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/BackgroundImage/Background.jpg);
background-repeat: no-repeat;
background-position: center;
background-size: cover;</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">background: transparent;
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="Resources.qrc">:/IconImage/Icon.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>48</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
color: #132440;
</string>
        </property>
        <property name="text">
         <string>Pass Paulito Manager</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
color: #132440;
</string>
        </property>
        <property name="text">
         <string>This program is a GSM CORPS property!           -Created by underpaid developer Paul</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
color: #132440;
</string>
        </property>
        <property name="text">
         <string>OPTIONS:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="Add_Pass_Manually_Button">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #16476A;
color: white;</string>
          </property>
          <property name="text">
           <string>Add Password Manually</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Add_Pass_Auto_Button">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #16476A;
color: white;</string>
          </property>
          <property name="text">
           <string>Add Password Automatically</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Show_All_Button">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #16476A;
color: white;</string>
          </property>
          <property name="text">
           <string>Show All Passwords</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Delete_Pass_Button">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #16476A;
color: white;</string>
          </property>
          <property name="text">
           <string>Delete a Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Update_Pass_Button">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background: #16476A;
color: white;</string>
          </property>
          <property name="text">
           <string>Update Existing Password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
color: #132440;
</string>
        </property>
        <property name="text">
         <string>*GSM CORPS does not guarantee against data breaches! HeeheeHawhaw</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
